### PR TITLE
Added required sections in Physics.continuum_mechanics and Physics.mechanics

### PR DESCRIPTION
**Pull request by Soumi7**
# Updates sympy.physics to follow latest docstring style guide

#### References to other Issues or PRs

#19591

#### Brief description of what is fixed or changed
- Required sections in :
  - sympy/physics/continuum_mechanics
      - beam.py
  - sympy/physics/hep
    - gamma_matrices.py
  - sympy/physics/mechanics/functions.py
  - sympy/physics/mechanics/kane.py
  - sympy/physics/mechanics/lagrange.py
  - sympy/physics/mechanics/linearize.py
  - sympy/physics/mechanics/models.py
  - sympy/physics/mechanics/particle.py
  - sympy/physics/mechanics/rigidbody.py

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by moorepants** [comment]
LGTM

**Comment by asmeurer** [comment]
Looks like there is a formatting error that causes the Sphinx build to fail

```
/home/travis/build/sympy/sympy/sympy/physics/hep/gamma_matrices.py:docstring of sympy.physics.hep.gamma_matrices.kahane_simplify:80: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

File: sympy/physics/hep/gamma_matrices.py [hunk 1]
```diff
@@ -340,7 +340,8 @@ def kahane_simplify(expression):
     Parameters
     ==========
 
-    `expression`    the tensor expression containing the gamma matrices to simplify.
+    ``expression``:
+        The tensor expression containing the gamma matrices to simplify.
```
Comment (asmeurer) [inline sympy/physics/hep/gamma_matrices.py:344]: The error is something in this docstring. I'm not sure what the problem is. Should there be a blank line here?

Comment (asmeurer) [reply sympy/physics/hep/gamma_matrices.py:344]: Oh perhaps the issue is there should be a space before the colon after ``` ``expression`` ```

Comment (Soumi7) [reply sympy/physics/hep/gamma_matrices.py:344]: Changed.

**Comment by moorepants** [comment]
This looks good to me if Travis passes.

**Comment by moorepants** [comment]
Thanks!

**Pull request merged**